Remove cui-wizard-form and cui-wizard-step

ganeshkumar1989 opened the pull request:
SCDC-774

### Requirements

* The cui-wizard-form and cui-wizard-step components have to be replaced by oui-stepper and oui-step-form from ovh-ui-angular

## Replace cui-wizard-form and cui-wizard-step with oui-stepper and oui-step-form 


### Description of the Change

The cui-wizard-form and cui-wizard-step components have been replaced with oui-stepper and oui-step-form from ovh-ui-angular

### Benefits

The component from the library is re-used, giving a more uniform look across Manager

marie-j commented on client/app/vps/additional-disk/order/vps-order-additional-disk.html:
```diff
@@ -2,60 +2,49 @@
-                                </label>
-                            </div>
+            <h1 class="oui-header_1">
+                {{ ::'vps_order_additionnal_disk_title' | translate }}
```
You can directly use `data-translate` on `<h1>`

ganeshkumar1989 replied:
This is fixed :)

marie-j commented on client/app/vps/additional-disk/order/vps-order-additional-disk.html:
```diff
@@ -2,60 +2,49 @@
+                    data-on-cancel="$ctrl.cancel()"
+                    data-on-submit="$ctrl.getAdditionalDiskFinalPrice()"
+                    data-valid="$ctrl.model.capacity">
+                    <div class="row-fluid" data-ng-repeat="capacity in $ctrl.capacityArray">
```
`track by $index`

ganeshkumar1989 replied:
This is done :)

marie-j commented on client/app/vps/backup-storage/order/vps-order-backup-storage.html:
```diff
@@ -2,42 +2,48 @@
-                             data-form-on-cancel="$ctrl.cancel()"
-                             data-form-on-complete="$ctrl.orderOption()">
+            <h2 class="oui-header_2">
+                {{ ::'vps_configuration_activate_ftpbackup_title' | translate }}
```
use `data-translate` instead

ganeshkumar1989 replied:
This is done :)

marie-j commented on client/app/vps/backup-storage/order/vps-order-backup-storage.html:
```diff
@@ -2,42 +2,48 @@
-                                 data-step-completed="true">
-                    <div class="text-center" data-ng-if="!$ctrl.model.optionDetails">
-                        <oui-spinner></oui-spinner>
+            <h3 class="oui-header_3" data-ng-bind-html="::'vps_configuration_activate_ftpbackup_step1_question' | translate:{ serviceName: $ctrl.serviceName }"></h3>
```
`data-translate` and `data-translate-values` instead of `data-ng-bind-html`

ganeshkumar1989 replied:
This is fixed :)

marie-j commented on client/app/vps/snapshot-order/vps-snapshot-order.html:
```diff
@@ -2,73 +2,62 @@
+                        data-model="$ctrl.model.contractsValidated">
+                    </oui-checkbox>
+                    <ul>
+                        <li data-ng-repeat="contract in $ctrl.model.optionDetails.contracts">
```
`track by`

ganeshkumar1989 replied:
This is done

marie-j commented on client/app/vps/translations/Messages_pt_PT.xml:
```diff
@@ -238,7 +238,7 @@
    <translation id="vps_configuration_activate_ftpbackup_title" qtlid="209866">Ativar a opção Backup storage</translation>
    <translation id="vps_configuration_activate_ftpbackup_step1_question" qtlid="452605">Deseja ativar a opção Backup Storage para o VPS <span class="normal italic">{{ serviceName }}</span>?</translation>
-   <translation id="vps_configuration_activate_ftpbackup_step1_alert" qtlid="452618">Este opção terá um preço mensal de <span class="red bold">{{ price }} IVA não incluído</span></translation>
+   <translation id="vps_configuration_activate_ftpbackup_step1_alert" qtlid="452618">Este opção terá um preço mensal de <span class="bold">{{ price }} IVA não incluído</span></translation>
```
Why change the portuguese translations instead of the english/french ones ?

ganeshkumar1989 replied:
According to requirement, the price in the this particular translation, should not be in red. It was red in all languages and this was removed for all languages.

cbourgois replied:
I think we should split the translation to avoid html tags in it.

marie-j commented on client/app/vps/veeam/order/vps-order-veeam.html:
```diff
@@ -2,71 +2,58 @@
+                    </oui-checkbox>
+                    <ul class="mb-4">
+                        <li data-ng-repeat="contract in model.optionDetails.contracts">
+                            <a target="_blank"
```
`rel="noopener"`

ganeshkumar1989 replied:
This is done for all external links

ganeshkumar1989 commented:
Using data-translate and data-translate-values instead of data-ng-bind= 'key' | translate and data-ng-bind-html in all applicable places
Added track by $index when using data-ng-repeat
Added rel="noopener" for external links

ganeshkumar1989 commented on client/app/vps/additional-disk/order/vps-order-additional-disk.html:
```diff
@@ -2,60 +2,47 @@
+                                <input type="radio"
+                                        data-ng-model="$ctrl.model.capacity"
+                                        name="capacity"
+                                        data-ng-value="{{::capacity.size}}"
```
This is done :)

ganeshkumar1989 commented on client/app/vps/additional-disk/order/vps-order-additional-disk.html:
```diff
@@ -2,60 +2,47 @@
+                    data-on-cancel="$ctrl.cancel()"
+                    data-on-submit="$ctrl.confirm()">
+                    <dl class="vps-dl-horizontal">
+                        <dt class="vps-item-horizontal bold" data-translate="vps_order_additional_disk_option"></dt>
```
This is done :)

ganeshkumar1989 commented on client/app/vps/additional-disk/order/vps-order-additional-disk.html:
```diff
@@ -2,60 +2,47 @@
 
     <div class="row d-lg-flex">
```
Both row and col-*-12 classes are removed

ganeshkumar1989 commented on client/app/vps/backup-storage/order/vps-order-backup-storage.html:
```diff
@@ -2,42 +2,45 @@
+                <oui-spinner></oui-spinner>
+            </div>
+            <div data-ng-if="$ctrl.model.optionDetails">
+                <div class="row-fluid">
```
This is done :)

ganeshkumar1989 commented on client/app/vps/backup-storage/order/vps-order-backup-storage.html:
```diff
@@ -2,42 +2,45 @@
+                    <p class="oui-paragraph" data-translate="vps_configuration_activate_ftpbackup_step1_alert" data-translate-values="{ price: $ctrl.model.optionDetails.unitaryPrice }"></p>
+                    <div class="oui-box mb-4">
+                        <p class="oui-paragraph" data-translate="vps_configuration_activate_ftpbackup_step1_confirmation"></p>
+                        <p class="bold">
```
This is done :)

ganeshkumar1989 commented on client/app/vps/snapshot-order/vps-snapshot-order.html:
```diff
@@ -2,73 +2,61 @@
+                            data-ng-bind="::$ctrl.model.optionDetails.withTax"></dd>
                     </dl>
-                    <div data-ng-show="$ctrl.model.url" class="row-fluid">
+                    <div data-ng-show="$ctrl.model.url" class="row-fluid mb-4">
```
row-fluid is removed

ganeshkumar1989 commented on client/app/vps/snapshot-order/vps-snapshot-order.html:
```diff
@@ -2,73 +2,61 @@
+                                data-ng-bind='contract.name'></a>
+                        </li>
+                    </ul>
+                    <div class="italic mb-4"
```
The equivalent for italic in bootstrap is <em> tag. Using this instead of italic class.

ganeshkumar1989 commented on client/app/vps/snapshot-order/vps-snapshot-order.html:
```diff
@@ -2,73 +2,61 @@
+                    <p data-translate="vps_configuration_activate_snapshot_step1_alert"
+                        data-translate-values="{ amount: $ctrl.model.optionDetails.unitaryPrice }"></p>
+                    <div data-translate="vps_configuration_activate_snapshot_step1_confirmation"></div>
+                    <div class="blue bold" data-translate="vps_configuration_veeam_order_step1_id"></div>
```
This is removed

ganeshkumar1989 commented on client/app/vps/snapshot-order/vps-snapshot-order.html:
```diff
@@ -2,73 +2,61 @@
+                    </oui-checkbox>
+                    <ul>
+                        <li data-ng-repeat="contract in $ctrl.model.optionDetails.contracts track by $index">
+                            <a target="_blank"
```
This is done :)

ganeshkumar1989 commented on client/app/vps/snapshot-order/vps-snapshot-order.html:
```diff
@@ -2,73 +2,61 @@
+                            <a target="_blank"
+                                rel="noopener"
+                                data-ng-href='{{::contract.url}}'
+                                data-ng-bind='contract.name'></a>
```
This is done :)

ganeshkumar1989 commented on client/app/vps/translations/Messages_cs_CZ.xml:
```diff
@@ -316,8 +316,8 @@
    <translation id="vps_configuration_veeam_order_step1_info" qtlid="169329">V rámci první platby Vám bude účtována doba přímo úměrná počtu dní do dalšího data obnovení VPS.</translation>
-   <translation id="vps_configuration_veeam_order_step1_info2" qtlid="452761">Služba bude následně fakturována na základě měsíční sazby <span class="red bold">{{ price }}</span></translation>
-   <translation id="vps_configuration_veeam_order_step1_question" qtlid="452774">Přejete si objednat automatické zálohy pro VPS <span class="italic normal">{{ serviceName }}</span>?</translation>
+   <translation id="vps_configuration_veeam_order_step1_info2" qtlid="452761">Služba bude následně fakturována na základě měsíční sazby <span class="bold">{{ price }}</span></translation>
```
Could you please point me to that? How can I update the translator? Any help here is appreciated

ganeshkumar1989 replied:
All translation files except Messages_fr_FR.xml have been reverted. Once merged with develop, the translation files for all other languages can be generated again.

ganeshkumar1989 commented on client/app/vps/snapshot-order/vps-snapshot-order.html:
```diff
@@ -2,73 +2,61 @@
+                        <li data-ng-repeat="contract in $ctrl.model.optionDetails.contracts track by $index">
+                            <a target="_blank"
+                                rel="noopener"
+                                data-ng-href='{{::contract.url}}'
```
This is fixed :)

ganeshkumar1989 commented:
All translation files except Messages_fr_FR.xml have been reverted. Once merged with develop, the translation files for all other languages can be generated again.

ganeshkumar1989 commented:
Only fr_FR file is modified for translation.
